lib/functions/rootfs/distro-agnostic.sh

```diff
@@ -152,9 +152,10 @@ function install_distribution_agnostic() {
 			DEFAULT_OVERLAYS=("${DEFAULT_OVERLAYS[@]/%/".dtbo"}")
 			DEFAULT_OVERLAYS=("${DEFAULT_OVERLAYS[@]/#/"${bootpart_prefix}dtb/${BOOT_FDT_FILE%%/*}/overlay/${OVERLAY_PREFIX}-"}")
 
-			display_alert "Adding to extlinux.conf" "fdtoverlays=${DEFAULT_OVERLAYS[@]}" "debug"
```

Comment:
Perhaps we can create new array names DEFAULT_OVERLAYS_ARR instead of changing DEFAULT_OVERLAYS. So we can keep if condition simple